dracut/modules.d/90wwinit/module-setup.sh

```diff
@@ -9,12 +9,12 @@ check() {
 }
 
 depends() {
-    echo network
+    echo network ignition
```

Comment:
Since ignition isn't available in Rocky Linux 8, we'd at least need a way to make this optional.